changes/pr238.yaml

```diff
@@ -0,0 +1,21 @@
+# An example changelog entry
+#
+# 1. Choose one (or more if a PR encompasses multiple changes) of the following headers:
+#   - feature
+#   - enhancement
+#   - fix
+#   - deprecation
+#   - breaking (for breaking changes)
+#   - migration (for database migrations)
+#
+# 2. Fill in one (or more) bullet points under the heading, describing the change.
+#    Markdown syntax may be used.
+#
+# 3. If you would like to be credited as helping with this release, add a
+#    contributor section with your name and github username.
+#
+# Here's an example of a PR that adds an enhancement
+
+enhancement:
+  - "Batch task and edge creation - [#238](https://github.com/PrefectHQ/server/pull/238)"
```

Comment:
```suggestion
  - "Batch task and edge insertion during flow creation - [#238](https://github.com/PrefectHQ/server/pull/238)"
```

Reply:
I would _consider_ adding a `fix` item as well i.e. "Allow flows with >1500 tasks to register without timeout"